### PR TITLE
Docs: Issue 78082 separate tips and tricks in user guide (3/3)

### PR DESCRIPTION
**Pull request by oraNod**
##### SUMMARY
Part three of #78082 that separates the tips and tricks in the user guide.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
Documentation: User Guide

##### ADDITIONAL INFORMATION
Creates a separate Tips and Tricks user guide.

### CODE REVIEW

File: docs/docsite/rst/tips_tricks/index.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Ansible tips and tricks
+#######################
+
+.. note::
+
+    **Making Open Source More Inclusive**
+
+    Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. We ask that you open an issue or pull request if you come upon a term that we have missed. For more details, see `our CTO Chris Wright's message <https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language>`_.
+
+Welcome to the Ansible tips and tricks guide for Microsoft Windows.
```
Comment (samccann) [inline docs/docsite/rst/tips_tricks/index.rst]:
```suggestion
Welcome to the Ansible tips and tricks guide.
```

File: docs/docsite/rst/user_guide/index.rst [hunk 2]
```diff
@@ -16,7 +16,6 @@ Quickly find answers in the following sections or expand the table of contents b
 Writing tasks, plays, and playbooks
 ===================================
 
-* I'm writing my first playbook. What should I :ref:`know before I begin <playbooks_tips_and_tricks>`?
 * I have a specific use case for a task or play:
```
Comment (samccann) [inline docs/docsite/rst/user_guide/index.rst:20]: Also need to pull tips and tricks out of the index in this file

Comment (oraNod) [reply docs/docsite/rst/user_guide/index.rst:20]: nice catch, thanks. I looked again and there were quite a few refs to tips and tricks + a duplicated rst page.

File: docs/docsite/rst/tips_tricks/index.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.. _tips_tricks_index:
+
```
Comment (samccann) [inline docs/docsite/rst/tips_tricks/index.rst:3]: Please always keep the old anchors around even if you want to add a new one like this.  While it's an extreme corner case, these anchors also work from a collection so if we remove it and a collection WAS using it, that's a broken link now.  See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation  - specifically the ``R()`` option.

File: docs/docsite/rst/tips_tricks/index.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
+.. _tips_tricks_index:
```
Comment (samccann) [inline docs/docsite/rst/tips_tricks/index.rst:1]:
```suggestion
.. _tips_tricks_index:
.. _playbooks_best_practices:
```

**Review by samccann — COMMENTED** [review]
I stumbled across an old docs issue about creating a user troubleshooting guide - https://github.com/ansible/ansible/issues/53457.

Makes me wonder if we should rename this troubleshooting instead of tips and tricks in the title?

File: docs/docsite/rst/tips_tricks/index.rst (hunk 3 above)
Comment (oraNod) [reply docs/docsite/rst/tips_tricks/index.rst:3]: Thanks for the heads up @samccann

**Comment by oraNod** [comment]
> I stumbled across an old docs issue about creating a user troubleshooting guide - #53457.
> 
> Makes me wonder if we should rename this troubleshooting instead of tips and tricks in the title?



> I stumbled across an old docs issue about creating a user troubleshooting guide - #53457.
> 
> Makes me wonder if we should rename this troubleshooting instead of tips and tricks in the title?

@samccann I had a look through that issue but don't feel like the tips and tricks content here falls into the realm of troubleshooting. It's more "these are some patterns that can help you use Ansible more effectively" rather than "these are some procedures you can follow when Ansible blows up".

That said I do think we could combine this page with some troubleshooting content. That would make sense alright. One of the reasons I broke this out into a separate thing is that the content spans playbooks, inventories, and execution plus general tips and tricks.

Maybe we could move some of the existing troubleshooting content in the UG, Gundalow mentions a couple files in that issue, as part of this PR. Then we could expand out in another PR with a view to resolving #53457

File: docs/docsite/rst/user_guide/windows_faq.rst [hunk 5]
```diff
@@ -3,4 +3,4 @@
 Windows Frequently Asked Questions
 ==================================
 
-This page has moved to :ref:`windows_faq`.
\ No newline at end of file
+This page has moved to :ref:`windows_faq`.
```
Comment (samccann) [inline docs/docsite/rst/user_guide/windows_faq.rst]: This and all the windows changes here look like a bad rebase happened? The windows user guide is already merged in devel so shouldn't be showing up here....https://docs.ansible.com/ansible-core/devel/user_guide/windows_dsc.html

Comment (oraNod) [reply docs/docsite/rst/user_guide/windows_faq.rst]: Weird. Yeah I'll fix this. Thanks for catching it.

**Review by samccann — APPROVED** [review]

**Pull request merged**